frontend/src/metabase/query_builder/components/view/sidebars/DatasetManagementSection/DatasetManagementSection.jsx

```diff
@@ -65,10 +56,6 @@ function DatasetManagementSection({
             <DatasetMetadataStrengthIndicator dataset={dataset} />
           </MetadataIndicatorContainer>
         </Row>
-        <Button
-          icon="model_framed"
-          onClick={turnDatasetIntoQuestion}
-        >{t`Turn back into a saved question`}</Button>
```

Comment:
`DatasetManagementSection` doesn't seem to be used at all now. I'd prefer removing it in another PR though because there's a chance we'd like to backport this one